apps/fluent-tester/src/TestComponents/Shadow/ShadowDepthTestSection.tsx

```diff
@@ -32,21 +24,13 @@ interface ShadowTestBoxProps {
 
 const ShadowTestBox: React.FunctionComponent<ShadowTestBoxProps> = (props: ShadowTestBoxProps) => {
   const theme = useTheme();
-  const themedStyles = getThemedStyles(theme);
   const backgroundColor = props.isBrand ? theme.colors.brandedBackground : theme.colors.background;
```

Comment:
I think I need to memoize backgroundColor here if we want to not cause extra re-renders?

Reply:
You could move it (and the viewStyle below) into getThemedStyles

Reply:
I need to props.isBrand though...haven't seen examples of this but I could make getThemedStyles take in an extra parameter?

Reply:
You could make two styles inside the stylesheet: one brand and one not

Reply:
Oooh

Reply:
Fixed, let me know if what I did was different from what you had in mind